app/src/Contacts.js

```diff
@@ -0,0 +1,147 @@
+import * as React from "react";
+
+import * as apiClient from "./apiClient";
+import "./global.module.scss";
+import Card from "./ui/Card";
+
+const Contacts = () => {
+  const [contacts, setContacts] = React.useState([]);
+  const [isAdding, setIsAdding] = React.useState(false);
+
+  const loadContacts = async () => setContacts(await apiClient.getContacts());
+  const addContact = (contact) =>
+    apiClient.addContact(contact).then(loadContacts).then(setIsAdding(false));
+
+  React.useEffect(() => {
+    loadContacts();
+  }, []);
+
+  return (
+    <section>
+      <ContactList {...{ contacts, loadContacts }} />
+      {!isAdding ? (
+        <button onClick={() => setIsAdding(!isAdding)}>Add a contact</button>
```

Comment:
[suggest] I would be explicit and do `setIsAdding(true)`, since we're not toggling but explicitly setting to the `true` state. Similar below when canceling, you're explicitly setting to `false`.